<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Whr</class>
 <widget class="QWidget" name="Whr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.057, y1:0.068, x2:1, y2:1, stop:0 rgba(255, 170, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>610</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>460</width>
     <height>610</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>460</width>
     <height>610</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.057, y1:0.068, x2:1, y2:1, stop:0 rgba(255, 170, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <widget class="QWidget" name="page">
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>610</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>610</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.057, y1:0.068, x2:1, y2:1, stop:0 rgba(255, 170, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font: 30pt &quot;MS Shell Dlg2&quot;;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Calculator WHR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
font:24px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Your gender: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
font:24px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Waist ratio [cm]:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
font:24px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Hip ratio [cm]:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">FONT-SIZE:24PX;
background-color: WHITE;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>300</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">FONT-SIZE:24PX;
background-color: WHITE;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">FONT-SIZE:24PX;
background-color: WHITE;</string>
     </property>
     <item>
      <property name="text">
       <string>MEN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WOMEN</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_calculate">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>420</y>
       <width>240</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 170, 0);
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_back_to_menu">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>530</y>
       <width>280</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
     </property>
     <property name="text">
      <string>Back to menu</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
